### DISCUSSION TITLE
When testing with modal true, jest test fails with "Cannot read properties of null (reading 'openPicker')"

### DISCUSSION THREAD

**Discussion by Ryan-Durling** [henninghall/react-native-date-picker#762] [General]
**Describe the bug**
When testing with jest and setting modal to true it throws an exception.
```
Cannot read properties of null (reading 'openPicker')
TypeError: Cannot read properties of null (reading 'openPicker')
    at DatePickerAndroid.openPicker [as render] 
```

**Expected behavior**

renders without error in jest test

**To Reproduce**
Add example code that reproduces the behavior. 
```javascript
it('does something') {
    render(<DatePicker
        modal={true}
        ... other props ...
      />)

        ... the rest of the test...
}
```

**Smartphone (please complete the following information):**
 - OS: [Android or iOS]
 - React Native version [e.g. 0.59.4]
 - react-native-date-picker version [e.g. 2.5.1]

**Comment by henninghall** [comment]
See https://github.com/henninghall/react-native-date-picker/discussions/423
